test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -450,15 +447,27 @@ TEST_F(ClusterManagerImplTest, InitializeOrder) {
   {
     "cds": {
       "cluster": {
-        "fake": ""
+        "name": "cds_cluster",
```

Comment:
Before, we validated the cluster schema in code that was mocked out. Now, we validate the schema  
 earlier in `parseClusterFromJson()` etc. So, we need something that will pass that validation.